Userland/Libraries/LibTLS/TLSv12.cpp

```diff
@@ -507,8 +507,9 @@ void TLSv12::pseudorandom_function(Bytes output, ReadonlyBytes secret, u8 const*
     }
 }
 
-TLSv12::TLSv12(StreamVariantType stream, Options options)
-    : m_stream(move(stream))
+TLSv12::TLSv12(StreamVariantType stream, Options options, PreventSIGPIPE prevent_sigpipe)
```

Comment:
imo there's no need to take this option, I'd personally turn it on for all sockets.